api/dimensions_test.go

```diff
@@ -15,8 +15,40 @@ import (
 	. "github.com/smartystreets/goconvey/convey"
 )
 
+// func to unmarshal and validate body bytes
+var validateBody = func(bytes []byte, expected models.DatasetDimensionResults) {
+	var response models.DatasetDimensionResults
+	err := json.Unmarshal(bytes, &response)
+	So(err, ShouldBeNil)
+	So(response, ShouldResemble, expected)
+}
+
 func TestGetDimensionsReturnsOk(t *testing.T) {
 	t.Parallel()
+
+	mongoData := []bson.M{
+		bson.M{"doc": bson.M{}},
+		bson.M{"doc": bson.M{
+			"links": bson.M{
+				"code_list": bson.M{},
+			},
+		},
```

Comment:
Think about collapsing lines 35 and 36 into 1 line `}},` that way the opening and closing of braces will have the same indentation

This makes it easier to spot mistakes when the incorrect number of braces are used